micronota/bfillings/tests/test_prodigal.py

```diff
@@ -22,8 +22,10 @@
 class ProdigalTests(TestCase):
     def setUp(self):
         self.temp_dir = tempfile.mkdtemp()
+        self.get_prodigal_path = partial(get_data_path,
+                                         subfolder='data/prodigal')
```

Comment:
:+1: for `join('data', 'prodigal')`

Could you clarify what your mean by "putting dir names into variables"?